setup.py

```diff
@@ -216,9 +216,7 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
 ]
 grpc = ['grpcio>=1.15.0']
 flask_oauth = [
```

Comment:
I think it should be renamed to "oauth" :)

Reply:
Can I change this name? It was named `flask_oauth`.